src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -169,30 +169,41 @@ uiModules.get('apps/management')
           notify.error('The file could not be processed.');
         }
 
-        return Promise.map(docs, (doc) => {
-          const { service } = find($scope.services, { type: doc._type }) || {};
-
-          if (!service) {
-            const msg = `Skipped import of "${doc._source.title}" (${doc._id})`;
-            const reason = `Invalid type: "${doc._type}"`;
-
-            notify.warning(`${msg}, ${reason}`, {
-              lifetime: 0,
-            });
-
-            return;
-          }
-
-          return service.get().then(function (obj) {
-            obj.id = doc._id;
-            return obj.applyESResp(doc).then(function () {
-              return obj.save({ confirmOverwrite : true });
-            });
-          });
+        return new Promise((resolve) => {
+          confirmModal(
+            `Would you like to overwrite any existing saved objects?`, {
```

Comment:
I went with "any" here because, at this point, the system doesn't know if it's going to find any conflicts. So what it's asking is, "if I find conflicts, would you like me to overwrite them?"

I'm guessing this isn't a one-word change... thoughts for asking that question better?